libraries/core-react/src/components/Table/index.tsx

```diff
@@ -1,19 +1,28 @@
-import { Table as BaseComponent, TableProps } from './Table'
+import * as React from 'react'
+import { FC } from 'react'
+import { Table as BaseTable, TableProps } from './Table'
 import { Body } from './Body'
 import { Cell } from './Cell'
 import { Head } from './Head'
 import { Row } from './Row'
 import { Caption } from './Caption'
+import { TableProvider } from './Table.context'
 
-type TableCompoundProps = typeof BaseComponent & {
+const TableWrapper: FC<TableProps> = (props) => (
```

Comment:
Does this still show up as `TableWrapper` in Storybook props table? Maybe change the const name or displayname to `Table`

Reply:
Hmm, yes. And still no props, forgot to check that